scipy/sparse/tests/test_base.py

```diff
@@ -3233,6 +3233,48 @@ def test_numpy_minmax(self):
             assert_array_equal(np.min(datsp), np.min(dat))
             assert_array_equal(np.max(datsp), np.max(dat))
 
+    def test_argmax(self):
+        D1 = np.array([
+            [-1, 5, 2, 3],
+            [0, 0, -1, -2],
+            [-1, -2, -3, -4],
+            [1, 2, 3, 4],
+            [1, 2, 0, 0],
+        ])
+        D2 = D1.transpose()
+
+        for D in [D1, D2]:
+            argmin = np.argmin(D)
+            argmax = np.argmax(D)
+            argmin_0 = np.argmin(D, axis=0)
+            argmax_0 = np.argmax(D, axis=0)
+            argmax_1 = np.argmax(D, axis=1)
+            argmin_1 = np.argmin(D, axis=1)
+
+            mat = self.spmatrix(D)
+
+            assert_equal(mat.argmax(), argmax)
```

Comment:
Nitpick: I think it's clearer to have tests of the form:

    assert_equal(mat.argmax(), np.argmax(D))

Rather than computing all the expected results first.